ui/src/Live/Multiview.tsx

```diff
@@ -150,14 +152,17 @@ function selectedReducer(old: SelectedCameras, op: SelectOp): SelectedCameras {
  * as possible.
  */
 const Multiview = (props: MultiviewProps) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
   const [selected, updateSelected] = useReducer(
     selectedReducer,
-    Array(MAX_CAMERAS).fill(null)
+    searchParams.has('cams') ? JSON.parse(searchParams.get('cams') || "") : Array(MAX_CAMERAS).fill(null)
```

Comment:
Do we need the `|| ""`? I still see the behavior change where the list show nothing instead of the `(none)` it used to, and suspect this is why.

Reply:
We need that, compiler throw error without that OR.
It's because searchParams.get return <string | null>, but json.parse accept only strings.